device/mellanox/x86_64-mlnx_msn4700-r0/sensors.conf.a1

```diff
@@ -0,0 +1,142 @@
+################################################################################
+# Copyright (c) 2019 Mellanox Technologies
```

Comment:
please change to
"Copyright (C) 20xx-2021 Mellanox Technologies, Ltd. ALL RIGHTS RESERVED."